src/host/CommandListPopup.cpp

```diff
@@ -294,7 +294,7 @@ void CommandListPopup::_cycleSelectionToMatchingCommands(COOKED_READ_DATA& cooke
         DWORD modifiers = 0;
 
         Status = _getUserInput(cookedReadData, popupKeys, modifiers, wch);
-        if (!NT_SUCCESS(Status))
+        if (!SUCCEEDED_NTSTATUS(Status))

```

Comment:
You could consider replacing all those `!SUCCEEDED_NTSTATUS` with `FAILED_NTSTATUS`.